paddle/CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ if(Boost_FOUND)
   add_subdirectory(memory)
   add_subdirectory(platform)
   add_subdirectory(framework)
+  add_subdirectory(operators)
```

Comment:
yes, there is a demo operator under operators, operator.h is still under paddle/framework